include/RegisterInfo.h

```diff
@@ -67,6 +67,13 @@ namespace ChimeraTK {
      */
     [[nodiscard]] std::vector<size_t> getQualifiedAsyncId() const;
 
+    /**
+     * Get the list of tags that are associated with this register.
+     *
+     * For a list of well-known tags, see TBD
```

Comment:
Did you leave the "TBD" in here intentionally?

Reply:
Yes because we do not have yet defined any.